bigip_plugin/bigip_cert.py

```diff
@@ -0,0 +1,144 @@
+########
+# Copyright (c) 2017 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import sys
+import os
+import uuid
+from shutil import copyfile
+
+from cloudify.decorators import operation
+from cloudify.exceptions import NonRecoverableError
+from cloudify.utils import exception_to_error_cause
+from cloudify import cluster, constants
+
+from bigip_sdk import bigip
+
+
+@operation
```

Comment:
@isaac-s I'm not sure how to test `bigip_cert.py` module and I I'm not sure who implement this part

What do you think about this to skip the changes you requested for this module to avoid breaking anything and to ask the one who did write this about how to test it